extensions/wikia/TriviaQuizzes/TriviaQuizzes.setup.php

```diff
@@ -1,24 +1,25 @@
 <?php
-$wgExtensionCredits['other'][] = [
+$wgExtensionCredits[ 'other' ][] = [
```

Comment:
We don't use spaces inside arrays in PHP, only inside parentheses.

Reply:
I should clarify, when assigning to an existing array, the spaces aren't used. I've look in other Wikia extensions and it's the same.

Reply:
I've seen both in different files (especially `.setup.php` are very inconsistent). I went with your suggestion.